src/components/ListItemLayout/styling/ListItemLayout.scss

```
@@ -0,0 +1,114 @@
+@use '../../../styling/utils';
+
+.str-chat__list-item-layout {
+  --list-item-padding: var(--str-chat__spacing-xs) var(--str-chat__spacing-sm);
+  display: flex;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)